### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by SJK9906** [Null-MC/Shrimple#80]
could add reflection an reflection of material (like molly VX) in molly VX you caa see reflection

molly VX
![Screenshot 2024-04-12 184331](https://github.com/Null-MC/Shrimple/assets/129201282/4106848f-1c8a-46cb-84c8-4fa12b66e806)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/a9a25e61-3978-4891-add9-9b872de4ef5c)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/1c048e78-966c-44c9-991c-e4a2d4b26b58)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/ad2a1e15-e2dc-4e04-8392-50046a28910a)

shrimple
![image](https://github.com/Null-MC/Shrimple/assets/129201282/b41a4cb2-f55a-42ea-9180-c678a638fac0)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/897edf10-2065-4e14-900e-b7d2e4dbdbbe)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/0a2eca85-d357-4d06-a459-ac2cd03692ba)
![image](https://github.com/Null-MC/Shrimple/assets/129201282/c6b1f57c-9e57-4750-9af9-a651ff33e17c)

**Comment by SJK9906** [comment]
Molly XV allow to have a multi reflected lray tracing
you can see light reflectid in a traslucient block like a mirror
more screenshot
molly VX
![image](https://github.com/Null-MC/Shrimple/assets/129201282/a1f084d8-af97-41b1-9013-c60c0990e9c1)

shrimple
<img width="959" alt="image" src="https://github.com/Null-MC/Shrimple/assets/129201282/a285f67a-b3f9-4312-9a43-98a955e24211">

resourspack: https://www.curseforge.com/minecraft/texture-packs/ultimacraft
config: [MollyVX-Shaders-All-Versions.zip.txt](https://github.com/Null-MC/Shrimple/files/14961673/MollyVX-Shaders-All-Versions.zip.txt)
shader loader: iris 1.6.17
minecraft 1.20.4

**Comment by null511** [comment]
nope, not gonna happen. I have simple screen-space reflections now like most shaders, world-space reflections require storing a _lot_ more data about the blocks that shrimple was just not designed for. It would take a complete rewrite and drastically reduce performance, so not planned.

**Comment by SJK9906** [comment]
what are some alternative shader with this suggestid foction?

**Comment by null511** [comment]
Rethinking Voxels

**Comment by xllifi** [comment]
> nope, not gonna happen. I have simple screen-space reflections now like most shaders, world-space reflections require storing a _lot_ more data about the blocks that shrimple was just not designed for. It would take a complete rewrite and drastically reduce performance, so not planned.

This means no RT reflections in this shader?

**Comment by null511** [comment]
> This means no RT reflections in this shader?

Correct; storing and tracing that data would ruin performance.
